tests/test_services.py

```diff
@@ -152,7 +152,7 @@ def test_edit_service_clear_values(self):
         initial_run_as = 'kenode'
         initial_trusted = False
         initial_type = ServiceType.NOTEBOOK
-        initial_env = ServiceEnvironmentVersion.PYTHON_3_7_NOTEBOOKS
+        initial_env = ServiceEnvironmentVersion.PYTHON_3_8_NOTEBOOKS
```

Comment:
this test fails in integration tests. Ensure that you assert both on `3.7 notebooks` and `3.8 notebooks` such that the test survives the changed backend defaults.

see: [line 182](https://github.com/KE-works/pykechain/pull/898/files#diff-caee650c3003842e9277d696a063ab724d42606e64cbf44cf1deb9d1d67d1d98R182)